src/NRediSearch/Client.cs

```diff
@@ -1,17 +1,18 @@
 ﻿// .NET port of https://github.com/RedisLabs/JRediSearch/
 
-using NRediSearch.Aggregation;
-using StackExchange.Redis;
 using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using NRediSearch.Aggregation;
+using StackExchange.Redis;
+using static NRediSearch.Schema;
 
 namespace NRediSearch
 {
     public sealed class Client
     {
         [Flags]
-        public enum IndexOptions
+        public enum IndexOption
```

Comment:
renaming a type is a breaking change; are there reasons? note: `[Flags]` enums are traditionally pluralized

Reply:
Fixed.